src/liquidity.rs

```diff
@@ -305,6 +310,79 @@ where
 		self.lsps2_client.as_ref().map(|s| (s.lsp_node_id, s.lsp_address.clone()))
 	}
 
+	pub(crate) fn lsps2_channel_needs_manual_broadcast(
+		&self, counterparty_node_id: PublicKey, user_channel_id: u128,
+	) -> Result<bool, APIError> {
```

Comment:
The `Result<.., APIError>` on `lightning-liquidity` pattern isn't great to begin with, but we should def. not have any methods in LDK Node using `APIError`. Please define an error variant if you really need one, but I suspect we don't actually need to.